docs/api/covidcast_times.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Date Coding and Revisions
+parent: COVIDcast Epidata API
+nav_order: 5
+---
+
+# Date Coding and Revisions
+
+Every observation in the COVIDcast Epidata API has two dates attached:
+
+* `time_value`: The time the underlying events happened. For example, for a data
+  source that reports on COVID test results, the time value is the date the
+  results were recorded by the testing provider.
+* `issue`: The date the estimates were *issued*. For example, a COVID test
+  result might be recorded on October 1st, but it may take several days for
+  that report to be collected, aggregated, received by Delphi, and added to our
+  database. The date Delphi makes the data available is the *issue date*.
+
+For example, consider using our [doctor visits signal](covidcast-signals/doctor-visits.md),
+which estimates the percentage of outpatient doctor visits that are
+COVID-related, and consider a result row with `time_value = "2020-05-01"` for
+`geo_value = "pa"`. This is an estimate for the percentage in Pennsylvania on
+May 1, 2020. That estimate was issued on May 5, 2020, the delay being due to the
```

Comment:
```suggestion
May 1, 2020. That estimate was issued on May 5, 2020, the delay is due to the
```

Reply:
This change creates a comma splice; we'd either have to make it two sentences or leave it as-is

Reply:
how about

```suggestion
May 1, 2020, which was issued on May 5, 2020. The delay is due to the
```